arranged mentor page

rishuraj1 opened the pull request:
## Related Issue

<!--
Provide information about the issue or bug that is being addressed.
-->

Closes: #514 

## Description of Changes

Changes:
- wrote fresh code keeping previous format
- utilized tailwind css to its fullest for better code readability.
- added: better styles and icons
- made the page fully responsive.

## Checklist:

<!--
Mark the checkboxes to indicate completion. Example:
- [x] My code follows the style guidelines of this project.
-->

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

<img width="415" alt="image" src="https://github.com/anmode/grabtern-frontend/assets/47839626/110351fa-53ca-4cdf-9e17-cb6190a46386">

![image](https://github.com/anmode/grabtern-frontend/assets/47839626/cf54c9f7-d341-4693-ac4a-134ebf446730)

![image](https://github.com/anmode/grabtern-frontend/assets/47839626/5729c6f6-eb0f-4187-9cb2-271253e8b44a)

![image](https://github.com/anmode/grabtern-frontend/assets/47839626/d1c08f5b-35ad-496e-9a6a-9c5e70f9b93a)

![image](https://github.com/anmode/grabtern-frontend/assets/47839626/b7c8259f-c42e-4892-8669-a01788e252ac)

anmode commented:
Just quick UI overview:
1. Instead blue Icon can we have our purple color/theme color 
2. FAQ are very big in size should be reduced..
3. Mobile optimization needed...
4. Please replace the vector(copyright issue), or change color to ours

have a look:https://mentro.tech/mentor/

rishuraj1 commented:
Hey @anmode, I have done the needed changes according to the above feedback,

- Forr the image/svg part, please have a look and tell me if this is what you were referring to. I have added a background of color purple to it.

- If the image is to be changed, can you please provide me with the image/vector to replace if you have already one.

- I have reduced the font sizes for the FAQ section with added responsiveness.
-  Made the page responsive for mobile devices.

<img width="960" alt="image" src="https://github.com/anmode/grabtern-frontend/assets/47839626/650f42e2-ac4f-4373-b582-d079ae16da00">

<img width="960" alt="image" src="https://github.com/anmode/grabtern-frontend/assets/47839626/c1c9cc66-9df0-491a-93d5-8ba6bfe5f76b">


dimensions: 410 x 722

<img width="248" alt="image" src="https://github.com/anmode/grabtern-frontend/assets/47839626/731b4725-b46a-4c46-a32e-05d0ae5aa7f4">

![image](https://github.com/anmode/grabtern-frontend/assets/47839626/0a8de6dd-a55a-4763-a693-7f905410a5f3)

![image](https://github.com/anmode/grabtern-frontend/assets/47839626/09be7de6-4a9d-44f5-8d80-3f942945791b)


Thank You !

anmode commented:
Some fixes:
1. Color code is wrong... this is our color: #845ec2
2. Theme image or main vector image is not good please replace or you can create awesome animation in other PR. It will of level 3 too. ( I'll provide vector then if you are not up for animation ) no worries :)
3. actully i was also talking about color of icon as cohort courses from to our color.

btw it is looking nice just needs some optimization.. let me know I'll set gmeet too if need

rishuraj1 commented:
Alright @anmode , I get it now, will make necessary changes. Can you please provide me the new image. I couldn't find one. Thanks

anmode commented:
Sorry @rishuraj1 i took time due to grab2.0 launch..
so here : https://www.freepik.com/free-vector/pensioners-financial-literacy-finance-education-savings-management-investment-awareness-consultant-explaining-finance-system-basics-elderly-people-vector-isolated-concept-metaphor-illustration_12083567.htm#query=mentorship&position=15&from_view=search&track=sph

anmode commented:
@rishuraj1 Need UX improvents too.. based on the page theme ( a mentor page to show what they will get from us you take refer from https://mentro.tech/mentor/, actually we have just copied same UX toh may be issue hoskta h.


2nd When I hover those services...cyan is coming.. change it to other color from this our theme pallet: ![b1c5e364-a2ad-40bb-b3ba-4101a0b05016](https://github.com/anmode/grabtern-frontend/assets/88332977/e564d057-9e7e-44ab-811d-3ed772d66638)

rishuraj1 commented:
Hey @anmode made some additional changes according to above theme pallete, I have changed the bg from cyan to #FBEAFF and used #845EC2 wherever needed. also I added another section "Become a Mentor" and some text under it so the page looks filled. I plan to add another section "testimonial" inspired from the reffered website above but, will open seperate issue for that. For now, Do let me know if any additional changes are required. Thank You !

anmode commented:
Wow looks cool, Hopefully this will be last fixes.. I'll explain in detail too... thanks for coperating
1. Remove the background of vector please
<img width="578" alt="Screenshot 2023-07-26 at 8 26 17 PM" src="https://github.com/anmode/grabtern-frontend/assets/88332977/1d59419b-34af-4a21-b67c-a6f8eca3d78c">

<img width="1224" alt="Screenshot 2023-07-26 at 8 27 28 PM" src="https://github.com/anmode/grabtern-frontend/assets/88332977/1d730aac-5663-4e52-a277-c700e97a43d0">

2.  In this section:
<img width="1221" alt="Screenshot 2023-07-26 at 8 27 48 PM" src="https://github.com/anmode/grabtern-frontend/assets/88332977/4c434cfe-cfec-46f8-a7a6-fd1b5f134872">

1. icon color before hover should be white
2. background color of rectangular box which is light pink should be replace by darker grey then background color.
3. On hover remove the cyan from icon instead you can have green from our pallet.

3. In this section:
<img width="1205" alt="Screenshot 2023-07-26 at 8 40 45 PM" src="https://github.com/anmode/grabtern-frontend/assets/88332977/a5b3e6c0-89af-48ee-a0ff-c13e375fdbbc">
some fixes as box should also close on clicking anywhere on screen too.

On open you see pink background color... same here replace by darker grey then background

rishuraj1 commented:
@anmode  , done with the changes. plz check

anmode commented:
ok let's break this issue...
please next PR need the mobile optimization and some UX fixes as faq and testimonials....
thanks for making it!

anmode commented on public/assets/index.js:
```diff
@@ -0,0 +1,6 @@
+export { default as MentorImg } from './img/MentorImg.png';
+export { default as BecomeMentor } from './img/BecomeMentor.png';
+export { default as GSoC } from './img/company/GSoC.svg';
+export { default as microsoft } from './img/company/microsoft.svg';
+export { default as mlh } from './img/company/mlh.svg';
+export { default as hackerRank } from './img/company/hackerRank.svg';
```
liked it!

anmode reviewed (approved):
cool
